File: client/src/app/Models/category.ts
export class Category {
    CategoryId: number;
    CategoryName: string;
    OrderCategory: number;

    /**
     *
     */
    constructor(CategoryId: number, CategoryName: string, OrderCategory: number) {
        this.CategoryId = CategoryId;
        this.CategoryName = CategoryName;
        this.OrderCategory = OrderCategory;

    }
}

File: API/Models/Category.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace API.Models
{
    public class Category
    {
        public int CategoryId { get; set; }
        public string CategoryName { get; set; }
        public int OrderCategory { get; set; }
        public virtual List<Product> Products { get; set; }
        
    }
}
